Source/Particles/Collision/BackgroundMCC/BackgroundMCCCollision.cpp

```diff
@@ -234,9 +234,12 @@ BackgroundMCCCollision::doCollisions (amrex::Real cur_time, amrex::Real dt, Mult
         // dt has to be small enough that a linear expansion of the collision
         // probability is sufficiently accurately, otherwise the MCC results
         // will be very heavily affected by small changes in the timestep
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(coll_n < 0.1_prt,
-            "dt is too large to ensure accurate MCC results"
-        );
+        if (coll_n < 0.1_prt) {
+            ablastr::warn_manager::WMRecordWarning("BackgroundMCC Collisions",
+                     "dt is too large to ensure accurate MCC results , coll_n: " +
+                      std::to_string(coll_n) + " is < 0.1 and collision probability is = " +
+                      std::to_string(m_total_collision_prob) + "\n");
```

Comment:
This should be as follows, right? The `WARPX_ALWAYS_ASSERT_WITH_MESSAGE` functionality is always confusing since it triggers when the condition is false i.e. `not coll_n < 0.1_prt` => `coll_n >= 0.1_prt`.
```suggestion
        if (coll_n > 0.1_prt) {
            ablastr::warn_manager::WMRecordWarning("BackgroundMCC Collisions",
                     "dt is too large to ensure accurate MCC results , coll_n: " +
                      std::to_string(coll_n) + " is > 0.1 and collision probability is = " +
                      std::to_string(m_total_collision_prob) + "\n");
```